src/R/Components/Impl/Plots/Implementation/NonScaledImage.cs

```diff
@@ -20,7 +20,7 @@ internal class NonScaledImage : Image {
                 // We prevent that scaling by calculating a size based on
                 // pixel size and dpi setting.
                 Size bitmapSize = new Size(bmp.PixelWidth, bmp.PixelHeight);
-                return WpfUnitsConversion.FromPixels(this, bitmapSize);
+                return WpfUnitsConversion.FromPixels(PresentationSource.FromVisual(this), bitmapSize);
```

Comment:
So, according to below, 
```
var source = PresentationSource.FromVisual(Content as Visual);
if (source != null) {
```
PresentationSource.FromVisual can still return null?